Friend/firmware/firmware_v1.0/prj_xiao_ble_sense_devkitv2-adafruit.conf

```diff
@@ -16,7 +16,7 @@ CONFIG_ADC=y
 
 CONFIG_BT=y
 CONFIG_BT_PERIPHERAL=y
-CONFIG_BT_DEVICE_NAME="Friend"
```

Comment:
Revert this change

Reply:
Francis told me to have it named Friend. Im getting mixed signals here?